gormschema/gorm.go

```diff
@@ -28,8 +27,9 @@ func New(dialect string, opts ...Option) *Loader {
 type (
 	// Loader is a Loader for gorm schema.
 	Loader struct {
-		dialect string
-		config  *gorm.Config
+		dialect            string
+		config             *gorm.Config
+		joinTableCallbacks []func(*gorm.DB)
```

Comment:
```suggestion
	modify func(*gorm.DB)
```

Reply:
I chose a specific name for this modifier because I'm not sure if in the future we will have another modifier that runs after db.AutoMigrate

Reply:
Let call it BeforeMigrate/AfterMigrate